src/app/games/checkers/international-checkers/InternationalCheckersTutorial.ts

```diff
@@ -0,0 +1,175 @@
+            CheckersState.of([
+                [_, _, _, _, _, _, _, _, _, V],
+                [_, _, _, _, _, _, U, _, V, _],
+                [_, _, _, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _, _, _],
+                [_, _, _, U, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _, _, _],
+            ], 2),
+            [
+                CheckersMove.fromStep(new Coord(6, 1), new Coord(5, 0)),
+                CheckersMove.fromStep(new Coord(6, 1), new Coord(7, 0)),
+            ],
+            TutorialStepMessage.CONGRATULATIONS(),
+            $localize`You did not choose the correct piece, and got no promotion.`,
+        ),
+        TutorialStep.fromPredicate(
+            $localize`Queen move`,
+            $localize`Queens can move forward like the others, and capture backward like the others. But they can also move backward without capturing. They are also able to do what we call "flying". This means doing longer steps, without jumping any piece to move, or by jumping exactly one opponent piece to capture it, then land wherever is wanting in the same line.<br/><br/>You're playing Dark, move your queen!`,
+            CheckersState.of([
+                [_, _, _, _, _, _, _, O, _, V],
+                [_, _, _, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, V, _, _],
+                [_, _, _, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _, _, _],
+                [_, _, U, _, _, U, _, U, _, U],
+            ], 2),
+            CheckersMove.fromStep(new Coord(7, 0), new Coord(0, 7)),
+            (move: CheckersMove, _ps: CheckersState, _rs: CheckersState) => {
+                if (move.getStartingCoord().equals(new Coord(7, 0))) {
+                    return MGPValidation.SUCCESS;
+                } else {
+                    return MGPValidation.failure($localize`You did not move your queen.`);
+                }
+            },
+            TutorialStepMessage.CONGRATULATIONS(),
+        ),
+        TutorialStep.anyMove(
+            $localize`Jump rule`,
+            $localize`When you do a multiple jump, you cannot jump twice over neither the same piece nor the same empty space.<br/>Here you have to apply all the different capturing rules: backward, maximal, flying, and of course not jumping twice over the same square.<br/><br/>You are playing Dark, go ahead.`,
```

Comment:
a multiple jump -> multiple jumps
ou multiple captures ?
et "jump rule" me semble pas bien nommé comme règle ? "La règle du saut" qui dit que "tu as pas le droit de passer deux fois sur la même case" ?! (y'a pas des noms standards des règles aux dames ?)

Reply:
puis, "not jumping twice over the same square": t'as pas non plus le droit d'aterrir sur une case sur laquelle tu es passée au dessus

Reply:
Que suggère tu ?